pkg/virt-api/webhooks/validating-webhook/admitters/vmexport-admitter_test.go

```diff
@@ -229,12 +230,32 @@ var _ = Describe("Validating VirtualMachineExport Admitter", func() {
 
 			ar := createExportAdmissionReview(export)
 			resp := createTestVMExportAdmitter(config).Admit(ar)
-			Expect(resp.Allowed).To(BeTrue())
+			Expect(resp.Allowed).To(BeTrue(), "should allow APIGroup: %s, Kind: %s", apiGroup, kind)
 		},
-			Entry("persistent volume claim", "v1", pvc),
-			Entry("virtual machine snapshot", kubevirtApiGroup, vmSnapshotKind),
+			Entry("persistent volume claim blank", "", pvc),
```

Comment:
can we also add a test case for `v1`? Thanks

Reply:
v1 is not an api group.... its a version, I need to adjust the message